src/API/Plugins.php

```diff
@@ -190,7 +190,7 @@ public function update_item_permissions_check( $request ) {
 	 * @return WP_Error|array Plugin Status
 	 */
 	public function install_plugin( $request ) {
-		$allowed_plugins = Onboarding::get_allowed_plugins();
+		$allowed_plugins = self::get_allowed_plugins();
 		$plugin          = sanitize_title_with_dashes( $request['plugin'] );
 		if ( ! in_array( $plugin, array_keys( $allowed_plugins ), true ) ) {
```

Comment:
I'm receiving an error trying to install or activate.  I think it's because the new array no longer uses the keys and instead includes an array of items:

```
(
    [0] => facebook-for-woocommerce
    [1] => mailchimp-for-woocommerce
    [2] => kliken-marketing-for-google
    [3] => jetpack
    [4] => woocommerce-services
    [5] => woocommerce-gateway-stripe
    [6] => woocommerce-gateway-paypal-express-checkout
    [7] => klarna-checkout-for-woocommerce
    [8] => klarna-payments-for-woocommerce
    [9] => woocommerce-square
    [10] => woocommerce-shipstation-integration
    [11] => woocommerce-payfast-gateway
    [12]=> woocommerce-payments
)
```